torchgeo/trainers/utils.py

```diff
@@ -18,7 +18,7 @@
 Conv2d.__module__ = "nn.Conv2d"
 
 
-def extract_encoder(path: str) -> Tuple[str, Dict[str, Tensor]]:
+def extract_encoder(path: str) -> Tuple[str, "OrderedDict[str, Tensor]"]:
```

Comment:
This is the `OrderedDict` hack I mentioned above. `model.load_state_dict` requires an `OrderedDict`, not `Dict`.